test/extended/deployments/deployments.go

```diff
@@ -782,4 +784,44 @@ var _ = g.Describe("deploymentconfigs", func() {
 			o.Expect(waitForLatestCondition(oc, name, deploymentRunTimeout, deploymentRunning)).NotTo(o.HaveOccurred())
 		})
 	})
+
+	g.Describe("ignores deployer and lets the config with a NewReplicationControllerCreated reason [Conformance]", func() {
+		g.AfterEach(func() {
+			failureTrap(oc, "database", g.CurrentGinkgoTestDescription().Failed)
+		})
+
+		g.It("should let the deployment config with a NewReplicationControllerCreated reason", func() {
+			_, name, err := createFixture(oc, ignoresDeployersFixture)
+			o.Expect(err).NotTo(o.HaveOccurred())
+
+			g.By("verifying that the deployment config is bumped to the first version")
+			err = wait.PollImmediate(500*time.Millisecond, 30*time.Second, func() (bool, error) {
+				dc, _, _, err := deploymentInfo(oc, name)
+				if err != nil {
+					return false, nil
+				}
+				return dc.Status.LatestVersion == 1, nil
+			})
+			if err == wait.ErrWaitTimeout {
+				err = fmt.Errorf("deployment config %q never incremented to the first version", name)
+			}
+			o.Expect(err).NotTo(o.HaveOccurred())
+
+			g.By("verifying that the deployment config has the desired condition and reason")
+			var conditions []deployapi.DeploymentCondition
+			err = wait.PollImmediate(500*time.Millisecond, 30*time.Second, func() (bool, error) {
+				dc, _, _, err := deploymentInfo(oc, name)
+				if err != nil {
+					return false, nil
+				}
+				conditions = dc.Status.Conditions
+				cond := deployutil.GetDeploymentCondition(dc.Status, deployapi.DeploymentProgressing)
+				return cond != nil && cond.Reason == deployutil.NewReplicationControllerReason, nil
+			})
+			if err == wait.ErrWaitTimeout {
+				err = fmt.Errorf("deployment config %q never updated its conditions: %#v", name, conditions)
+			}
+			o.Expect(err).NotTo(o.HaveOccurred())
```

Comment:
as some point (and we are getting to that point now) we should split this file :-)